jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/JibPluginConfigurationTest.java

```diff
@@ -40,18 +39,14 @@
 
   private static final MavenProject project = new MavenProject();
```

Comment:
Oh didn't see that it was a static MavenProject before. And I feel like this might be bad practice since now we're responsible for cleaning it up between each test. If we don't have a great reason to keep this static, could we change it to just `private final MavenProject` ?